src/sundials/sundials_profiler.c

```diff
@@ -548,16 +535,17 @@ int sunCompareTimes(const void* l, const void* r)
   const SUNHashMapKeyValue left  = *((SUNHashMapKeyValue*)l);
   const SUNHashMapKeyValue right = *((SUNHashMapKeyValue*)r);
 
-  if (left == NULL && right == NULL) return (0);
-  if (left == NULL) return (1);
-  if (right == NULL) return (-1);
+  if (left == NULL && right == NULL) { return SUN_SUCCESS; }
```

Comment:
Should this really be `SUN_SUCCESS` instead of `0`?  it seems like this function returns 1,0,-1 to indicate the direction of the comparison operation, not a success/failure in the comparison.  As such, I think that `0` is much more understandable.

Reply:
Agreed, it should be 0.